Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocValidationErrorIndicatorColumnDefinitionTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Moq;
+using NUnit.Framework;
+using Remotion.Development.UnitTesting;
+using Remotion.ObjectBinding.Web.UI.Controls;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls
+{
+  [TestFixture]
+  public class BocValidationErrorIndicatorColumnDefinitionTest
+  {
+    [Test]
+    public void GetComparer_ReturnsInstanceOfBocListRowWithValidationFailureComparer ()
+    {
+      var column = new BocValidationErrorIndicatorColumnDefinition();
+
+      var failureRepository = new Mock<IBocListValidationFailureRepository>();
+      var bocListStub = new Mock<IBocList>();
+      bocListStub.Setup(_ => _.ValidationFailureRepository).Returns(failureRepository.Object);
+      column.OwnerControl = bocListStub.Object;
+
+      var comparer = column.CreateCellValueComparer();
+      Assert.That(comparer, Is.InstanceOf<BocListRowWithValidationFailureComparer>());
+      Assert.That(PrivateInvoke.GetNonPublicField(comparer, "_validationFailureRepository"), Is.EqualTo(failureRepository.Object));
```

Comment:
Stylistic: I try to use Is.SameAs when I assert objects that are intended as references instead of values.